superset-frontend/cypress-base/cypress/integration/dashboard_list/card_view.test.ts

```diff
@@ -84,16 +84,16 @@ describe('Dashboard card view', () => {
     cy.get('.ant-card').last().contains("World Bank's Data");
   });
 
-  it('should delete correctly', () => {
+  it.only('should delete correctly', () => {
```

Comment:
This `only` should probably be removed, right?

Reply:
Oops, removed